cluster/environments/azure-multiple-clusters/README.md

```diff
@@ -6,16 +6,17 @@ The `azure-multiple-cluster` environment deploys three redundant clusters (simil
 
 This deployment creates the following:
 
-- [Three different AKS clusters](#cluster-deployment)
-- [A Public IP Address](#public-ip-address) for each cluster
-- [Azure Traffic Manager](#traffic-manager-deployment)
+- [azure-multiple-cluster](#azure-multiple-cluster)
+  - [Getting Started](#getting-started)
+    - [Cluster Deployment](#cluster-deployment)
+    - [Public IP Address](#public-ip-address)
+    - [Traffic Manager Deployment](#traffic-manager-deployment)
 
-You can deploy the `azure-multiple-cluster` using a Service Principal that either has or does not have `Owner` privileges on the Azure Subscription using the variable `service_principal_is_owner`.  When set to `1`, `Owner` privileges are required and the Public IP for each AKS cluster will be deployed into the Resource Group specified for each of the clusters (see [Cluster Deployment](#cluster-deployment)).  When set to `0`, the Public IP for each AKS cluster will be provisioned in the Resource Group generated by the AKS Cluster Provisioner.  In this second case, `Owner` privileges are not required.  The reason for these options is based on allowing the AKS Cluster to make use of the Public IP, which is discussed [here](https://docs.microsoft.com/en-us/azure/aks/static-ip).  The default behavior is to deploy the cluster requiring `Owner` privileges.
+You can deploy the `azure-multiple-cluster` using a Service Principal that either has or does not have `Owner` privileges on the Azure Subscription using the variable `service_principal_is_owner`.  The Public IP for each AKS cluster will be provisioned in the Resource Group generated by the AKS Cluster Provisioner. 
```

Comment:
`service_principal_is_owner` has been removed.  So the public IP will be deployed to the appropriate, non-derived resource group.  Owner level privileges are required.